src/OrleansServiceFabricUtils/FabricServiceSiloResolver.cs

```diff
@@ -132,14 +133,6 @@ private void OnPartitionChange(long handlerId, FabricPartitionResolutionChange a
                     if (!existing.IsSamePartitionAs(updated.Partition)) continue;
                     found = true;
 
-                    if (updated.Partition.IsOlderThan(existing))
-                    {
-                        this.log.Info($"Update for partition {updated} is superseded by existing version.");
-
-                        // Do not update the partition if the exiting one has a newer version than the update.
-                        break;
```

Comment:
Not really, I was just finding that these checks really don't add value. In the worst case - that there's some race and updates are out of order, we have stale information for 1 polling cycle (5 seconds).